makefiles/Makefile.devhub

```diff
@@ -1,6 +1,8 @@
 GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
 USER=$(shell whoami)
 
+DEDICATED_BUCKET=true
```

Comment:
Hey saw this and we actually use `devhub-content` and `devhub-content-integration`. Not sure where this job came from but it isn't being used currently. If you don't mind I'll comment a few things that may be off in this PR?